docs/patching.rst

```diff
@@ -30,19 +30,17 @@ The Quick and Dirty
 ===================
 
 Very quick, very little explanation. Those with strong git fu may already see
-some shortcuts. Use them! As long as ``mozilla/master`` doesn't have merge
-commits, it's all good.
+some shortcuts. Use them!
```

Comment:
"Those who know git well will probably see ways they can take shortcuts to make these commands easier." Should I reword this? It was already here.